burn-tch/build.rs

```diff
@@ -0,0 +1,15 @@
+use std::env;
+fn main() {
+    // Temporary workaround for https://github.com/burn-rs/burn/issues/180
+    // Remove this once tch-rs upgrades to Torch 2.0 at least
+
+    if cfg!(all(target_arch = "aarch64", target_os = "macos")) {
+        let message = "Run scripts/fix-tch-build-arm64.py to fix environment variables for torch";
```

Comment:
I would add the link to the issue in the comment, so it's clearer why this step is necessary.

Reply:
And I guess most poeple using burn from crates.io won't have access to the script, so pointing to the link online is important.